### ISSUE TITLE
Add docker and commit data to healtcheck route

### ISSUE THREAD

**Issue by john-g-g** [steemit/condenser#2809]
The healthcheck route `.well-known/healthcheck.json` currently just displays `{"status":"ok"}`. Most other steemit services also include the docker tag and git source commit sha in their responses, and condenser should too. The improved response should be:
```
{"status":"ok","source_commit":"whatever","docker_tag":"whatever"}
```
The value for the docker tag should be read from the `DOCKER_TAG` env var, and source commit should be read from the `SOURCE_COMMIT` env var.

**Comment by bnchdrff** [comment]
not working in prod

**Comment by jredbeard** [comment]
closed by #2873
